### PR TITLE
trace file that is returning null errors and disable fonts folder embedded on the executable + add setup file

### PR DESCRIPTION
**Pull request by crowplexus**
TL;DR - useful for debugging and embedding the fonts path is really not needed anymore, as psych has custom font support on mods

---------------------------------------

this PR enables file outputs on files that are "returning null NOOOO"
output should be something like
```
oh no NOTE_assets.xml is returning null NOOOO
```
(NOTE_assets is an example)

the fonts folder was embedded by default on the base game, Psych however, has support for custom fonts, this might be handy in case you wanna change, say, the FPS Counter with source code using a custom font for example, since the fonts folder will be generated once the game is compiled, then you could just do something like ``Paths.font("pixel.otf")`` for example

### CODE REVIEW

**Comment by adamsaustria** [comment]
it is still doing this error

**Comment by BeastlyGhost** [comment]
> it is still doing this error

this is not to fix the error, because it's not an error in the first place, it is simply to print the **file** that is returning the error

**Review by Raltyro — APPROVED** [review]

**Review by rodney528 — APPROVED** [review]

**Review by JuniorNovoa1 — APPROVED** [review]
fucking please

**Review by dummmmdum — APPROVED** [review]

**Review by MemeHoovy — APPROVED** [review]

**Review by rodney528 — APPROVED** [review]

**Review by JorosTheModderPH — APPROVED** [review]

File: source/Paths.hx [hunk 1]
```diff
@@ -355,7 +355,7 @@ class Paths
 			localTrackedAssets.push(path);
 			return currentTrackedAssets.get(path);
 		}
-		trace('oh no its returning null NOOOO');
+		trace('oh no ' + key + ' is returning null NOOOO');
```
Comment (Itz-Miles) [inline source/Paths.hx:358]: numbers can't explain how ingenious this single addition is, but words can.

approved/10

**Review by GO-Miles — APPROVED** [review]

**Comment by BeastlyGhost** [comment]
will reopen later maybe idk
